src/frontend/src/Components/map/mapPopup.scss

```diff
@@ -213,10 +224,13 @@
         &__title {
           .name {
             color: #CE3E39;
+            font-size: 24px;
```

Comment:
We should be using rem conversions for font-size for accessibility purposes (so that the browser setting for bigger/smaller font size can take in effect).

Reply:
https://github.com/bcgov/DriveBC.ca/commit/f23af461e5a378e22f396396384179cffe502317